src/openms/source/FORMAT/MSstatsFile.cpp

```diff
@@ -727,49 +727,30 @@ bool OpenMS::MSstatsFile::isQuantifyable_(
     const std::set<OpenMS::String>& accs,
     const std::unordered_map<OpenMS::String, const OpenMS::IndProtGrp*>& accession_to_group) const
 {
-  bool quantifyable = true;
+  if (accs.empty()) return false;
 
-  if (accs.empty())
-  {
-    quantifyable = false;
-  }
-  else if (accs.size() > 1)
-      // every prot accession in the set needs to belong to the same indist. group to make this peptide
-      // eligible for quantification
+  if (accs.size() == 1) return true;
```

Comment:
I don't think this is true. If this accession is not part of the given, considered proteins it should not be quantified.

Reply:
Unless you add a check/precondition that all accessions in evidences occur in the map (which is built from the first run only).

Reply:
Another thing that we probably should define at the beginning of the function, is if every protein in this first run needs to be part of a group (i.e. all singletons are there) or not. I just realized that this also heavily affects the logic.

Reply:
I think PLFQ makes sure this is the case. I think we should still add a consistency check if data is consistent with the inference run.

Reply:
I will add an issue as this has some higher priority